src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -862,6 +865,17 @@ void RNN::prepareParams() {
 
     prim = result.first;
 
+    auto pd = (*prim).get_primitive_desc();
```

Comment:
I would agree. Lets move this block in common place as DNNL utility. We will need it once decide to enable scratchpad for other DNNL primitives.

Reply:
changed